blueocean-rest/src/main/java/io/jenkins/blueocean/Routable.java

```diff
@@ -12,7 +10,7 @@
  */
 public interface Routable {
     /**
-     * Follows the same convention as {@link Action#getUrlName()}
+     * Follows the same convention as Action.getUrlName()
```

Comment:
Did not want to add `Action` to whitelist as nothing depends on it except for javadoc.